wrangler-core/src/main/java/io/cdap/functions/Global.java

```diff
@@ -213,4 +213,74 @@ public static String unquote(String string) {
     }
     return string;
   }
+
+  /**
+   * Returns true when an expression does not evaluate to the null value.
+   *
+   * @param value to be evaluated.
+   * @return true when not null, false otherwise.
+   */
+  public static boolean IsNotNull(Object value) {
+    if (value != null) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * Returns true when an expression evaluates to the null value.
+   *
+   * @param value to be evaluated.
+   * @return false when not null, true otherwise.
+   */
+  public static boolean IsNull(Object value) {
+    return !IsNotNull(value);
+  }
+
+  /**
+   * Returns an empty string if the input column is null, otherwise returns the input column value.
+   *
+   * @param value to be evaluated.
+   * @return Empty string if null, else 'value'
+   */
+  public static Object NullToEmpty(Object value) {
+    if (IsNull(value)) {
+      return new String("");
+    }
+    return value;
+  }
+
+  /**
+   * Returns zero if the input column is null, otherwise returns the input column value.
+   *
+   * @param value to be evaluated.
+   * @return Empty string if null, else 'value'
+   */
+  public static Object NullToZero(Object value) {
+    if (IsNull(value)) {
+      return 0;
+    }
+    return value;
+  }
+
+  /**
+   * Returns the specified value if the input column is null, otherwise returns the input column value.
+   *
+   * @param value to evaluated.
+   * @param replace value to replace with.
+   * @return value if not null, else with replaced value.
+   */
+  public static Object NullToValue(Object value, Object replace) {
+    if (IsNull(value)) {
+      return replace;
+    }
+    return value;
+  }
+
+  /**
+   * @return Assigns a null value to the target column.
+   */
+  public static Object SetNull() {
```

Comment:
I don't see a test for this, but would the user use this like `set-column col SetNull()`? Seems weird, couldn't you just do `set-column col null`?

Reply:
Yea `set-column col null` will have same effect. This is just to simulate what data stage has. I have added a test for it.

Reply:
let's remove this, I don't think it makes sense to have it.